gandalf/src/main/java/io/github/btkelly/gandalf/checker/DefaultHistoryChecker.java

```diff
@@ -36,7 +37,10 @@
     private final SharedPreferences prefs;
 
     public DefaultHistoryChecker(@NonNull final Context context) {
-        prefs = PreferenceManager.getDefaultSharedPreferences(context);
+        prefs = context.getSharedPreferences(
+                context.getPackageName() + "_preferences",
+                Context.MODE_PRIVATE
+        );
```

Comment:
❓ is this potentially breaking?

Reply:
It shouldn't be as that is the code of the method I replaced. I needed to replace it for unit testing as Android overrides the method calls for things to return default values. In this case it was always returning null from a static method which would not allow me to mock the response.

Reply:
Ok cool 👍 :D